src/Symfony/Component/Notifier/Bridge/OrangeSms/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "symfony/orange-sms-notifier",
+    "type": "symfony-notifier-bridge",
+    "description": "Symfony Orange Sms Notifier Bridge",
+    "keywords": [
+        "sms",
+        "orange-smsapi",
+        "orange",
+        "notifier"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Fabien Potencier",
+            "email": "fabien@symfony.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
```

Comment:
Please use the required version for Symfony 6

Reply:
It's done.

Reply:
Should I then change the branch of the pull request?

Reply:
Yes please.